### PR TITLE
Updates for JWST Cycle 2 Call for Proposals

### PR DESCRIPTION
**Pull request by mfixstsci**
- Update JWST ephemeris
- Update dates in scripts and tests
- Remove old ephemeris

### CODE REVIEW

**Comment by mfixstsci** [comment]
Screen Shot 2022-10-31 at 1 58 37 PM
Screen Shot 2022-10-31 at 1 55 31 PM
Screen Shot 2022-10-31 at 1 54 48 PM
Screen Shot 2022-10-31 at 1 53 24 PM
Screen Shot 2022-10-31 at 1 49 21 PM

**Comment by mfixstsci** [comment]
@PatrickOgle here are the changes to the GTVT/MTVT. Attached are figures from the examples listed on the README. For some of the moving target, Horizons may have changed the way you can access those specific designations. I think we remove them from the docs. We have a few working examples of the GTVT and MTVT and can provide a link to other designations in the Horizons database. If you think this is a good idea, I can make those changes in this PR.

**Review by bholler — CHANGES_REQUESTED** [review]
Small change requested: In bin/jwst_gtvt, the latest date is listed as 2024-10-03 and everywhere else it is 2024-10-02. The latest date in the updated file is 2024-10-03. I also just checked Horizons and the JWST ephemeris now goes out to 2024-10-31.

I'm fine leaving the end date at 2024-10-03 (it includes all of Cycle 2 and part of Cycle 3), but we should be consistent with the latest advertised date.

**Comment by mfixstsci** [comment]
@bholler thanks for catching that. The GTVT goes up to `n-1` of the rows in the ephemeris. If you try running it with the last date in the ephemeris, it will crash with an index out of range error.

**Review by bholler — APPROVED** [review]
That's what I thought, so changing the end date in bin/jwst_gtvt to 2024-10-02 is the right (and easiest) choice. Looks good now!

**Comment by PatrickOgle** [comment]
@mfixstsci I will rely on you to select any examples and figures to update in JDox. The existing ones look fine to me, even if they are for an earlier ephemeris and date range..

**Comment by mfixstsci** [comment]
Thanks @PatrickOgle I think that keeping the package data and figures up-to-date makes sense but if we want to update jdox later, we can do that. I just don't want to cause any confusion with that on the user end which is why I suggested it.

**Comment by mfixstsci** [comment]
@PatrickOgle @bholler I think this is ready to go if one of you would like to review.

**Comment by bholler** [comment]
@mfixstsci I downloaded the latest version from your branch and tested multiple use cases and flags. All of them worked as expected. The default dates are correct (2021-12-26 to 2024-10-02) for the plots and are correctly presented in the help text for the jwst_gtvt and jwst_mtvt.

There may be updates to the JWST ephemeris coming within the next month, but they would likely be too late to support a release ahead of the Cycle 2 CfP.

I will next work on updating the GTVT and MTVT JDOX pages and writing short release notes for the 0.3.0 G/MTVT release.

**Comment by mfixstsci** [comment]
@PatrickOgle Will you potentially have time to review this today? If we want to add the background support by the time the cycle 2 CFP, I would like to have this working version in this week and some time to work on the backgrounds PR this week and next. Thank you!

File: README.md [hunk 1]
```diff
@@ -78,11 +78,9 @@ For moving targets, use `jwst_mtvt`
 
 Periodic comets and most asteroids benefit from using the `--smallbody` flag
 
-`$ jwst_mtvt 2`  # Venus (no windows)
+`$ jwst_mtvt 7`  # Uranus (no windows)
```
Comment (mfixstsci) [inline README.md:81]: The whole point was to not get data and show the `--smallbody`

**Comment by mfixstsci** [comment]
@bholler could you please give me a review/approval and then we can merge this in and release it? Thank you!

**Pull request merged**